src/program/lwaftr/doc/CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # Change Log
 
+## [2.3] - 2016-02-17
+
+A bug fix and performance improvement release.
+
+ * Fix case in which TTL of ICMPv4 packets was not always being
+   decremented.
+
+ * Fix memory leaks when dropping packets due to 0 TTL, failed binding
+   table lookup, or other errors that might cause ICMP error replies.
+
+ * Fix hairpinning of ICMP error messages for non-existent IPv4 hosts.
+   Before, these errors always were going out the public IPv4 interface
+   instead of being hairpinned if needed.
+
+ * Fix hairpinning of ICMP error messages for incoming IPv4 packets
+   whose TTL is 0 or 1. Before, these errors always were going out the
+   public IPv4 interface instead of being hairpinned if needed.
+
+ * Fix hairpinning of ICMP error messages for packets with the DF bit
+   that would cause fragmentation. Likewise these were always going out
+   the public interface.
+
+ * Allow B4s that have access to port 0 on their IPv4 address to be
+   pinged from the internet or from a hairpinned B4, and to reply.  This
+   enables a B4 with a whole IPv4 address to be pinged.  Having any
+   reserved ports on an IPv4 address will prevent pings.
```

Comment:
It's worth clarifying that those hosts can still initiate pings and get replies, just not be pinged themselves.